configs/distill/mmcls/byot/byot_logits_resnet18_cifar100_8xb16_in1k.py

```diff
@@ -0,0 +1,155 @@
+_base_ = [
```

Comment:
what's the purpose of `_in1k.py` and `_in1k_kl.py`?

Reply:
In gml config the soft label loss is KDsoftCE, while in paper KL.
Experiment shows a similar accuracy (top-1:80.63->80.66, top-5:95.57->95.76).
Now in `_in1k.py` KL is applied.